### PR TITLE
Documentation/2022 12 21 minutes

### PR DESCRIPTION
**Pull request by JesusSierralaya**
please review or fill missing parts of today´s meeting

### CODE REVIEW

File: doc/minutes/2022-12-21.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+#### Tasks
+
+-   **Roberto**: review what remains of the draft
+
+-   **Daniel**: make the PR and merge
+
+-   **Everybody**: Must approve the previous changes
+
+## About what we need to put about ourselves
```
Comment (robertomediavilla) [inline doc/minutes/2022-12-21.md]: I suggest rename this section "On the team description section"

**Review by robertomediavilla — DISMISSED** [review]
Thanks Jesús. I just added a minor suggestion but won't request changes

**Review by Cristrinaranjus — DISMISSED** [review]

**Comment by DaniMori** [comment]
@JesusSierralaya I can't request a review from your side, but you can "review and comment" anyway

File: doc/minutes/2022-12-21.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+-   **Everyone**: Approve changes in the corresponding Github PRs [ASAP]
+
+-   **Roberto**: Submit form to Yerun Open Science Awards call [21/12/2022]
+
+## Team description section
+
+#### Debate
+
+-   Suggested that the "team description"" would better include a description of
```
Comment (robertomediavilla) [inline doc/minutes/2022-12-21.md]: "team description", one " extra

**Review by robertomediavilla — DISMISSED** [review]
Thaaaanks!!

**Review by DaniMori — COMMENTED** [review]
@JesusSierralaya it seems like I branched from `drafting/description` instead of `main`. Hopefully it doesn't create any conflict...

**Review by JesusSierralaya — COMMENTED** [review]
I approve the previous changes!

**Review by Cristrinaranjus — DISMISSED** [review]

File: doc/minutes/2022-12-21.md (hunk 2 above)
Comment (DaniMori) [reply doc/minutes/2022-12-21.md]: Thanks! Fixed in baf61bb8cfc2219a1ff0dfff6198db5a1844edd4

**Review by DaniMori — APPROVED** [review]

**Review by Cristrinaranjus — APPROVED** [review]

**Comment by DaniMori** [comment]
@robertomediavilla and @Leafrancia please review and approve latest version, thanks!

**Review by Leafrancia — APPROVED** [review]

**Review by robertomediavilla — APPROVED** [review]

**Pull request merged**